backend/models/recipientSchema.js

```diff
@@ -1,14 +1,15 @@
-const mongoose = require('mongoose')
+const mongoose = require("mongoose");
 
 const RecipientSchema = new mongoose.Schema(
   {
     name: String,
     EntityType: String,
-    DemographicName: String,
```

Comment:
These schema changes were already made on `sprint-refactor`, I think you may need to pull from that branch. Try running `git pull origin sprint-refactor` to pull those changes in. That should apply all of the schema modifications. 

(Also just in general, whenever you're making a pr it's good practice to frequently pull in the changes from that branch to stay up to date and minimize the number of merge conflicts you get)